### ISSUE TITLE
64-bit NVDA - Copy from virtual buffer does not work

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#18967]
### Steps to reproduce
* Open a webpage in a browser, e.g. Firefox or Chrome
* Select some text
* Press `control+c` to copy it

### Actual behavior
An error is logged and the text is not copied.
### Expected behavior
The text should be copied.
### NVDA logs, crash dumps and other attachments
```
IO - inputCore.InputManager.executeGesture (09:10:25.761) - winInputHook (21604):
Input: kb(desktop):control+c
ERROR - scriptHandler.executeScript (09:10:25.762) - MainThread (5840):
error executing script: <bound method CursorManager.script_copyToClipboard of <NVDAObjects.IAccessible.chromium.ChromeVBuf object at 0x0000014793C95810>> with gesture 'contrôle+c'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 300, in executeScript
  File "cursorManager.pyc", line 543, in script_copyToClipboard
  File "textInfos\__init__.pyc", line 612, in copyToClipboard
  File "api.pyc", line 411, in copyToClip
  File "winUser.pyc", line 890, in setClipboardData
  File "contextlib.pyc", line 148, in __exit__
  File "winKernel.pyc", line 549, in lock
ctypes.ArgumentError: argument 1: TypeError: 'HGLOBAL' object cannot be interpreted as an integer

```
#### NVDA installed/portable/running from source
Temp copy from installer
#### NVDA version
nvda_snapshot_try-64bit-52706,70587e82.exe
#### Windows version
Windows 11 24H2 (AMD64) build 26100.6584
#### Name and version of other software in use when reproducing the issue
* Firefox: '143.0.1'
* Chrome: unknown for now (latest stable)
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors
Last alpha (nvda_snapshot_alpha-52705,dc226976.exe), which is 32-bit, is correctly working.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by michaelDCurran** [comment]
I believe this is when calling GlobalUnlock from kernel32.dll. Our ctypes defintion in winBindings.kernel32 is correct. argtypes is `(HGLOBAL,)` and restype is BOOL.
However, when running this build and looking with the NvDA Python console, I see that `winBindings.kernel32.GlobalUnlock.argtypes` is `[ctypes.c_long]`.

This is not what we set, nor is it at all correct, especially for 64 bit.

Something, some dependency is changing GlobalUnlock's ctypes defintion underneath us.

This makes it again more important now that we should isolate our winBindings definitions so that  global changes from dependencies do not affect them, and nor will our winBindings definitions affect theirs.

**Comment by michaelDCurran** [comment]
Once pr #18956 is merged I should be able to do this conversion work.
